package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -307,7 +307,14 @@ class HELANAL(AnalysisBase):
     flatten_single_helix : bool, optional
         Whether to flatten results if only one selection is passed.
     split_residue_sequences : bool, optional
-        Wether to split residues into a list of same or consecutive helices.
+        Whether to split residues into a list of same or consecutive helices.
+        If False, the residues will be analysed as a single helix.
+        Holds True, if more than one `select` string given Otherwise False.
+        i.e., if len(select) > 1, then split_residue_sequences = True
+        example-1: if select=['protein and name CA', 'protein and name CA']
+                   then split_residue_sequences = True
+        example-2: if select=['protein and name CA']
+                   then split_residue_sequences = False
```

Comment:
Have you checked if this is actually true? There's an example of how to use HELANAL at the user guide here: https://userguide.mdanalysis.org/stable/examples/analysis/structure/helanal.html which you can download as a Jupyter notebook here: https://github.com/MDAnalysis/UserGuide/blob/develop/doc/source/examples/analysis/structure/helanal.ipynb

Maybe try passing in your example `select` sequence and see if it has the effect described here.

Reply:
> Have you checked if this is actually true? There's an example of how to use HELANAL at the user guide here: https://userguide.mdanalysis.org/stable/examples/analysis/structure/helanal.html which you can download as a Jupyter notebook here: https://github.com/MDAnalysis/UserGuide/blob/develop/doc/source/examples/analysis/structure/helanal.ipynb
> 
> Maybe try passing in your example `select` sequence and see if it has the effect described here.

Hi, I have written this actually according to test examples shared by @PicoCentauri in this  ->https://github.com/MDAnalysis/mdanalysis/issues/3313#issuecomment-1099137937

Reply:
It's always wise to test something works instead assuming it -- even core developers are not always right, or have the perfect solution! We do not want to have incorrect documentation as that will be frustrating for users. When you write code or documentation you should always try to verify that it works for yourself, instead of just trusting that it will.

Reply:
oh yeah alright! I will test it for sure!

Reply:
oh yeah alright! I will test it for sure!

Reply:
Oh yeah ok alright! I will test it for sure !

Reply:
Oh yeah ok alright! I will test it for sure !

Reply:
@lilyminium i think i need to change the statements for this `split_residues_sequences`. I ran the code in the notebook & i got to know that this `split_residues_sequences` will come into consideration only when len(groups) > 1 ie when `selection` has gaps in the residues. ie. if we have `selection = ['resid 6:50 or resid 100:130 or resid 132:148']` then only the `len(groups)>1` and then only we consider the passed value of this parameter `split_residues_sequences` . ie if the passed value for this parameter is True then it will split the residue having gaps otherwise will not split it.
as can be seen in the code  --> 
![image](https://user-images.githubusercontent.com/36916536/163724999-5bb4ddc2-6b01-47d3-9305-6b238f526162.png)

Reply:
can you please confirm my finding so that i can make changes in the commit accordingly.

Reply:
Yeah, that's right -- good job figuring that out! It would be ideal if in the documentation, you explained what kind of `select` string might lead to `len(groups) > 1`, as `groups` is not passed into the analysis but the user themselves.

Reply:
Sorry I didn't get you? What kind of select string means??